packages/static-wado-scp/lib/dicomWebScpConfig.js

```diff
@@ -13,8 +13,7 @@ const { dicomWebScpConfig } = ConfigPoint.register({
     isGroup: true,
     helpShort: "dicomwebscp",
     helpDescription: "Creates server to receive data on DIMSE and store it DICOM",
-    scpAe: "DICOMWEB",
-    scpPort: 11112,
+    queryAe: "DICOMWEB",
```

Comment:
This should go in a custom config file, eg tests/static-wado-remote.json5 would work

Reply:
Comments to Rodolfo: Use command line to get the config file